CHANGELOG.md

```diff
@@ -1,123 +1,210 @@
 # Changelog
 
-## main
+All notable changes to this project will be documented in this file.
 
-### heroku-jvm-application-deployer
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
 
 * Add proper CLI to configure all aspects of app deployment. Previously used Java properties are no longer supported. See `--help` for usage. ([#232](https://github.com/heroku/heroku-maven-plugin/pull/232))
 * Unify usage between WAR and JAR files. heroku-jvm-application-deployer will now automatically use the correct mode based on file extension. ([#232](https://github.com/heroku/heroku-maven-plugin/pull/232))
 * Default `webapp-runner` version is now always the most recently released version. ([#232](https://github.com/heroku/heroku-maven-plugin/pull/232))
 
-## 3.0.5
+## [3.0.7] - 2023-02-06
+
+### Changed 
+
+* Update dependencies.
+
+## [3.0.6] - 2022-12-01
+
+### Changed
+
+* Update dependencies.
+
+## [3.0.5] - 2022-10-26
+
+### Changed
 
 * Update dependencies. ([#142](https://github.com/heroku/heroku-maven-plugin/pull/142))
 
-## 3.0.4
+## [3.0.4] - 2020-08-11
+
+### Fixed
 
 * Fix `heroku:deploy-war` goal. ([#77](https://github.com/heroku/heroku-maven-plugin/pull/77))
 
-## 3.0.3
+## [3.0.3] 2020-07-09
```

Comment:
This heading and a few of the ones below are missing the hyphen between the version number and the date.

Reply:
Thanks, fixed! :)